app/src/components/NavBar.tsx

```diff
@@ -2,17 +2,16 @@ import React from 'react';
 
 import { Alignment, Box, Button, Direction, Image, PaddingSize, Spacing, Stack } from '@kibalabs/ui-react';
 
-import { useAccountId, useOnLinkAccountsClicked } from '../AccountContext';
+import { useAccount, useOnLinkAccountsClicked } from '../AccountContext';
 import { Account } from './Account';
 
 export const NavBar = (): React.ReactElement => {
-  const accountId = useAccountId();
+  const accountId = useAccount();
```

Comment:
this is wrong - useAccount gives an Account not an accountId. this shouldnt work. have you run it locally?